### DISCUSSION TITLE
FlyoutPage navigation

### DISCUSSION THREAD

**Discussion by lightway82** [PrismLibrary/Prism#2537] [Xamarin.Forms]
I use Prism 8.1.
I have root flyout page and open main page  from App.cd as:
```
 protected override async void OnInitialized()
        {
            InitializeComponent();        
            await NavigationService.NavigateAsync("/RootPage/NavigationPage/MainPage");            
        }
````

I call   ```await NavigationService.NavigateAsync("ResultPage", navigationParams);``` from MainpageViewModel and I see page without hamburger button, but I can show flyout menu by swipe.

![image](https://user-images.githubusercontent.com/17110240/129895513-c6219ed5-6089-4293-b244-76039b3dbd39.png)

If I call ```await NavigationService.NavigateAsync("NavigationPage/ResultPage", navigationParams);``` , then I see two navigation bar: one with hamburger button and one with back button.

![image](https://user-images.githubusercontent.com/17110240/129894970-8f8f98e4-2a28-406a-b295-9d89fcb5fe5a.png)

I need page which appearing after navigation with hamburger button and navigation button(arrow). Is it  possible?

**Comment by brianlagunas** [comment]
That's exactly how a flyout page works. You don't get a hamburger icon when you navigate within a NavPage. You get a back arrow.

**Reply by lightway82** [reply]
Thanks
